lib/astc_codec.cpp

```diff
@@ -981,11 +1188,13 @@ static void decompression_workload_runner(int thread_count, int thread_id, void*
 /**
  * @ingroup reader
  * @brief Decodes a ktx2 texture object, if it is ASTC encoded.
+
+ * The decompressed format is calculated from coressponding ASTC format. There are
+ * only 3 possible options currently supported. RGBA8, sRGBA8 and RGBA32.
```

Comment:
Use SRGBA as that corresponds to the spelling in the VkFormats.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/967/commits/b7acc0aa7d4b45c8286a341dae73bdd8101fe21c